HousingSearchApi.Tests/V1/Helper/SearchPhraseTests.cs

```diff
@@ -21,7 +24,21 @@ public SearchPhraseTests()
         [Theory]
         [InlineData("")]
         [InlineData(null)]
-        public void ShouldReturnNullIfRequestSearchTextIsEmpty(string searchText)
+        public void ShouldReturnFilterOnlyByTypeIfRequestSearchTextIsEmpty(string searchText)
```

Comment:
Is there a reason why we no longer wish this to be null?

Reply:
We'he changed search logic. Now we search by text and by type, and type is required. So if input text is empty - we will search only by type